syncano/models/options.py

```diff
@@ -95,7 +96,7 @@ def add_field(self, field):
             raise SyncanoValueError('Field "{0}" already defined'.format(field.name))
 
         self.field_names.append(field.name)
-        self.fields.append(field)
+        self.fields.insert(bisect(self.fields, field), field)
```

Comment:
As far as I know, it's very useful in cli for example, when cli app is prompting user for input.